src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -2472,4 +2472,41 @@
     </Environment>
   </Provider>
 
+  <!--
+                                                ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                                ██░▄▄░█░▄▄█░▄▄▀█░▄▄█░▄▄█░██░█░▄▄
+                                                ██░█▀▀█░▄▄█░██░█░▄▄█▄▄▀█░▀▀░█▄▄▀
+                                                ██░▀▀▄█▄▄▄█▄██▄█▄▄▄█▄▄▄█▀▀▀▄█▄▄▄
+                                                ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <Provider
+    Name="Genesys"
+    DisplayName="Genesys Cloud"
+    Id="2e34b9d0-5c1f-4e23-9ac9-b4aefde213d2"
+    Documentation="https://developer.genesys.cloud/">
```

Comment:
A more specific documentation page - ideally dedicated to OAuth 2.0 - would be better.